README.md

```diff
@@ -129,7 +130,57 @@ noisy log only displayed when verbose is enabled
 
 In addition to the constructor, `roosevelt-logger` exposes the following properties:
```

Comment:
In the previous version of the logger, these properties were static, so you would do the following to get them:

```js
const rooseveltLogger = require('roosevelt-logger')
rooseveltLogger.winston
```

but now they are instance variables of the class:

```js
const rooseveltLogger = require('roosevelt-logger')
const logger = new rooseveltLogger()
logger.winston
```

With this, if you create different instances of the logger, they will have their own scoped version of the internal Winston logger.

So it may be good to say that these are instance methods / instance variables on the class instances rather than a more static instance.